mtp_bank_admin/assets-src/stylesheets/app.scss

```diff
@@ -46,3 +46,19 @@
     margin-bottom: 3em;
   }
 }
+
+.mtp-missed-downloads-box {
+  background: file-url('icon-important.png') no-repeat 16px 16px;
+  border: 3px solid $red;
+  padding: 1em;
+  margin-bottom: 2em;
+
+  @include device-pixel-ratio {
+    background-image: file-url('icon-important-2x.png');
```

Comment:
you need a `background-size` c.f. https://github.com/ministryofjustice/money-to-prisoners-common/blob/master/mtp_common/assets-src/stylesheets/elements/_delete-icons.scss